File: exercises/04_exercise_for_loops/05_salary.py
num_open_tabs = int(input())
salary_amount = int(input())
left_money = 0
for number in range(1, num_open_tabs + 1):
    tab = input()
    if tab == "Facebook":
        salary_amount -= 150
    elif tab == "Instagram":
        salary_amount -= 100
    elif tab == "Reddit":
        salary_amount -= 50
    if salary_amount == 0:
        break
if salary_amount <= 0:
    print(f"You have lost your salary.")
elif salary_amount > 0:
    print(f"{salary_amount:2d}")

File: 05_JavaScriptBasics/02_Exercises/04_ForLoop/05_salary.js
function salary (input) {
    let index = 0;
    let openedTabs = Number(input[index]);
    index++;
    let salary = Number(input[index]);
    index++;

    for (let i = 1; i <= openedTabs; i++) {
       let name = input[index];
       index++;

       switch (name) {
            case "Facebook":
               salary -= 150;
            break;
            case "Instagram":
                salary -= 100;
            break;
            case "Reddit": 
                salary -= 50;    
            break;
       }
    }

    if (salary <= 0) {
        console.log("You have lost your salary.");
    } else {
        console.log(Math.abs(salary));
    }
}

salary(["10", "750", "Facebook", "Dev.bg", "Instagram", "Facebook", "Reddit", "Facebook", "Facebook"]);
salary(["3", "500", "Github.com", "Stackoverflow.com", "softuni.bg"]);
salary(["3", "500", "Facebook", "Stackoverflow.com", "softuni.bg"]);
